mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This is a bridge module between Management System and Management System
```

Comment:
@kaynnan Can you be more specific?

Reply:
I made changes to the description, I believe it is now more detailed